setup.py

```diff
@@ -36,6 +36,7 @@
     "packaging >= 20.0.0",
     "pandas>=1.1.0",
     "pyarrow >= 3.0.0",
+    "pydata-google-auth >=1.5.0",
```

Comment:
Please also update the minimum supported Python's corresponding constraints file to make sure this minimum version continues to be accurate.

Per https://github.com/googleapis/python-bigquery-magics/blob/943d0a4a8850ef182d493915651cd99523f52213/setup.py#L114, our minimum Python version is 3.7. Therefore, we need to include this dependency in https://github.com/googleapis/python-bigquery-magics/blob/main/testing/constraints-3.7.txt

Reply:
Done!